test/api/model/model_checks.dart

```diff
@@ -1,6 +1,14 @@
 import 'package:checks/checks.dart';
+import 'package:zulip/api/model/initial_snapshot.dart';
 import 'package:zulip/api/model/model.dart';
 
+extension RealmDefaultExternalAccountChecks on Subject<RealmDefaultExternalAccount> {
```

Comment:
nit: put these in the `foo_checks.dart` file corresponding to the `foo.dart` the class is defined in